packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.js

```diff
@@ -0,0 +1,78 @@
+import counterpart from 'counterpart'
+import { Box, Grid, Paragraph } from 'grommet'
+import { array, bool, number, string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import en from './locales/en'
+import ContentBox from '../../../../shared/components/ContentBox'
+import SubjectPreview from '../../../../shared/components/SubjectPreview'
+
+counterpart.registerTranslations('en', en)
+
+function RecentSubjects (props) {
+  const { isLoggedIn, recents, projectName, size, slug } = props
+  const height = (size === 1) ? '40vw' : '200px'
+
+  return (
+    <ContentBox title={counterpart('RecentSubjects.title', { projectName })}>
+      <Paragraph margin={{ top: 'none' }}>
+        {counterpart('RecentSubjects.text')}
+      </Paragraph>
+      <Grid
+        alignContent='stretch'
+        columns={[`repeat(${size}, 1fr)`]}
+        gap='small'
+      >
+        {recents.map(recent => {
+          const subject = {
+            favorite: recent.favorite,
+            id: recent.subjectId,
+            locations: recent.locations,
+            toggleFavourite: recent.toggleFavourite
+          }
+          return (
+            <SubjectPreview
+              height={height}
+              key={recent.subjectId}
+              isLoggedIn={isLoggedIn}
+              subject={subject}
+              slug={slug}
+              width={'100%'}
+            />
+          )
+        })}
+        {[...Array(size - recents.length)].map((placeholder, i) => {
+          return (
+            <Box
+              align='center'
+              justify='center'
+              height={height}
+              key={i}
+              overflow="hidden"
+              width={'100%'}
+            >
+              <img alt="" role='presentation' src='/static/subject-placeholder.png' />
+            </Box>
```

Comment:
Non-blocking: I would suggest pulling this into a placeholder component and also using it with the `Media` component (it supports a `placeholder` prop that you can define a React component or HTML), so we have consistency in placeholders being used. Right now this is shown if there aren't enough recents, but I don't see any reason to also not use this as the placeholder while the image is being processed by the thumbnail service.

Reply:
The Media component has issues with image sizing but I can give this a go.

Reply:
Sorry, are you suggesting that this image also be used as a placeholder in the `SubjectPreview` component, or that this `<img>` element be replaced with a `Media` component?

Reply:
Also noting here so that it isn't lost: `RecentSubjects` still isn't tested.

Reply:
The `Media` placeholder has the same sizing bug that was fixed in #1160 for video and audio. You can see it here, where the placeholder is rendered at 700x700px (the maximum dimensions of the thumbnail), rather than the dimensions of the supplied image.

Reply:
Tracked in #1195